shared/reducers/wallets.tsx

```diff
@@ -436,7 +436,8 @@ export default function(state: Types.State = initialState, action: WalletsGen.Ac
     case WalletsGen.updateAirdropBannerState:
       return state.merge({airdropShowBanner: action.payload.show})
     case WalletsGen.updatedAirdropDetails:
-      return state.merge({airdropDetails: action.payload.details})
+      const {details, isPromoted} = action.payload
+      return state.set('airdropDetails', Constants.makeAirdropDetails({details, isPromoted}))
```

Comment:
How ofter does this happen? If a lot, seems we could use some comparison here, something like

```
      return state.update('airdropDetails', airdropDetails =>
        airdropDetails
          .set('isPromoted', isPromoted)
          .update('details', oldDetails => oldDetails.equals(details) ? oldDetails : details)
      )

```

If not frequent, then I'm OK with this too!

Reply:
Ah, looks like just once at app start, so I'll leave it as-is.